source/extensions/filters/http/jwt_authn/jwks_async_fetcher.cc

```diff
@@ -70,7 +70,10 @@ void JwksAsyncFetcher::handleFetchDone() {
 void JwksAsyncFetcher::onJwksSuccess(google::jwt_verify::JwksPtr&& jwks) {
   stats_.jwks_fetch_success_.inc();
 
+  ENVOY_LOG(warn, "{}: succeeded", debug_name_);
```

Comment:
It should not be  "warn" log,  maybe "info" if not frequented

Reply:
done.